hawc/apps/common/exports.py

```diff
@@ -152,7 +152,7 @@ def format_time(self, df: pd.DataFrame) -> pd.DataFrame:
         tz = timezone.get_default_timezone()
         for key in [self.get_column_name("created"), self.get_column_name("last_updated")]:
             if key in df.columns:
-                df.loc[:, key] = df[key].dt.tz_convert(tz).dt.strftime("%Y-%m-%dT%H:%M:%S.%f%z")
+                df[key] = df[key].dt.tz_convert(tz).dt.strftime("%Y-%m-%dT%H:%M:%S.%f%z")
```

Comment:
@shapiromatron discovered the reason `format_time` not working is due to pandas update to 2.1.1 in PR #928. In [this update, ](https://pandas.pydata.org/docs/whatsnew/v2.0.0.html): 'Changed behavior in setting values with df.loc[:, foo] = bar or df.iloc[:, foo] = bar, these now always attempt to set values inplace before falling back to casting ([GH 45333](https://github.com/pandas-dev/pandas/issues/45333))'.

Reply:
I can create a ticket for that